kubetest/boskos/common/common.go

```diff
@@ -116,7 +116,7 @@ type Resource struct {
 type ResourceEntry struct {
 	Type     string        `json:"type"`
 	State    string        `json:"state"`
-	Names    []string      `json:"names,flow"`
+	Names    []string      `json:"names"`
```

Comment:
Seems we got really lucky with this one, the go json encoder still respects the first value and just ignores the `flow`